study.md

```diff
@@ -0,0 +1,2 @@
+# esleint
```

Comment:
typo here.
Can you explain the purpose of this file?

Reply:
sorry, committed  by mistake.